### DISCUSSION TITLE
Okta Integration

### DISCUSSION THREAD

**Discussion by Swampcritter** [YOURLS/YOURLS#2894] [Plugins]
Is there any possibility of an Okta Integration for YOURLS?

https://developer.okta.com/product/authentication/

**Comment by ozh** [comment]
I'm sure there is and that it's [Possible with a plugin](https://github.com/YOURLS/YOURLS/wiki/Possible-With-A-Plugin)

There are already several plugins implementing other authentication protocols, they may be a good inspiration to hack from.
